vhdbuilder/packer/install-dependencies.sh

```diff
@@ -211,6 +211,15 @@ ENV{DEVTYPE}=="partition", ENV{AZURE_DISK_TYPE}=="?*", ENV{AZURE_DISK_NAME}=="?*
 ENV{DEVTYPE}=="partition", ENV{AZURE_DISK_TYPE}=="?*", ENV{AZURE_DISK_SERIAL}=="?*", SYMLINK+="disk/azure/\$env{AZURE_DISK_TYPE}/by-serial/\$env{AZURE_DISK_SERIAL}-part%n"
 LABEL="azure_disk_end"
 EOF
+
+cat > /etc/udev/rules.d/99-microsoft-mana-mtu.rules <<EOF
+# Udev rule to set MTU to 9000 for Microsoft MANA Ethernet controllers
+# This rule triggers when a network interface is added and checks for Microsoft Azure Network Adapter VF
+# https://learn.microsoft.com/en-us/azure/virtual-network/how-to-virtual-machine-mtu?tabs=linux
+
+SUBSYSTEM=="net", KERNEL=="en*", ENV{ID_NET_DRIVER}=="mana", RUN+="/usr/sbin/ip link set dev eth0 mtu 9000"
```

Comment:
what doe this `KERNEL=="en*"` cover ?

Reply:
the physical Nic on accelerated network (SR-IOV) enabled VM, for example enP30832s1, and eth0 is the virtual nic that OS and application interact with